index.js

```diff
@@ -48,17 +72,18 @@ CompassCompilerPlugin.prototype.toTree = function(tree, inputPath, outputPath, i
     var outputFileName = path.basename(outputPaths[file], extension) + extension; // new name for asset
     var outputDir = removeLeadingSlash(path.dirname(outputPaths[file])); // new directory for asset
     var node = new Funnel(compassTree, {
-      srcDir: outputPath,
```

Comment:
Because `srcDir` should be the source directory of the Compass compiler's output tree, not the `outputPath` within the compass output tree that may or may not exist.

If `srcDir` is set to `outputPath`, Ember CLI throws a `ENOENT` error when trying to compile `addon.scss` because the intermediate Compass output file actually lives at `COMPASS_OUTPUT_TREE/addon.css` instead of `COMPASS_OUTPUT_TREE/assets/addon.css`.

This goes the same for the test dummy app: `COMPASS_OUTPUT_TREE/app.css` and `COMPASS_OUTPUT_TREE/test-support.css`, **not:** `COMPASS_OUTPUT_TREE/asset/app.css` and `COMPASS_OUTPUT_TREE/asset/test-support.css`.

Can you tell me how it breaks your app and also the test environment (Ember CLI version, Node.js version, etc.)?